### PR TITLE
fix loading on java 8 

### PR DESCRIPTION
**Pull request by InsaneLegacy**
fixes issue #10

### CODE REVIEW

File: src/main/java/com/filiphsandstrom/mineiago/Logging.java [hunk 1]
```diff
@@ -29,19 +27,18 @@ public Logging() {
...
+    
     private void Print(String msg) {
         if(MineiaGo.getInstance().getConfig().getLoglevel() >= 3)
             return;
-
+            
         String file_content;
         PrintWriter printer;
         try {
-            file_content = Files.readString(loggingFile.toPath(), StandardCharsets.UTF_8);
```
Comment (InsaneLegacy) [inline src/main/java/com/filiphsandstrom/mineiago/Logging.java]: this whas causing the error it should work now on java 8

**Comment by filfat** [comment]
Without filecontent the log will be overwritten with the lastest log out every time.

We need to keep the history.


Also can you squash your commits into one?

**Comment by InsaneLegacy** [comment]
@filfat am unable to fix this in other way but now you know what problem is

**Comment by InsaneLegacy** [comment]
squash commit no idera how do that do

**Comment by filfat** [comment]
Can you try “file_content = new String(Files.readAllBytes(loggingFile.toPath()));”?

**Comment by filfat** [comment]
Should be fixed in a recent commit.

**Pull request closed without merging**